examples/counter_functional/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "counter_functional"
```

Comment:
This package needs to be added to the workspace root [`/Cargo.toml`](https://github.com/yewstack/yew/blob/master/Cargo.toml).